pkg/features/features.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2021 The Kruise Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package features
+
+import (
+	"k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/component-base/featuregate"
+
+	utilfeature "github.com/openkruise/agents/pkg/utils/feature"
+)
+
+const (
+	// SandboxGate enable Sandbox-controller to create a sandbox pod.
+	SandboxGate featuregate.Feature = "Sandbox"
+
+	// SandboxSetGate enable SandboxSet-controller to create a sandbox pod.
+	SandboxSetGate featuregate.Feature = "SandboxSet"
+)
+
+var defaultFeatureGates = map[featuregate.Feature]featuregate.FeatureSpec{
+	SandboxGate:    {Default: true, PreRelease: featuregate.Alpha},
+	SandboxSetGate: {Default: true, PreRelease: featuregate.Alpha},
```

Comment:
sandbox and sandboxset is the core controllers, it is not desirable to turn these feature off, instead i think there will be use cases where the platform users only need the sandbox controller not the sandbox manager.